Gemfile

```diff
@@ -4,6 +4,8 @@ source 'https://rubygems.org'
 gem 'rails', '4.2.6'
 gem 'devise'
 gem 'simple_form'
+gem 'rmagick'
+gem 'carrierwave', github: 'carrierwaveuploader/carrierwave'
```

Comment:
@ianroberts131 The `github: ..` option will mean that the gem version installed will not be a stable release version. Instead it will use that latest from the carrierwave GitHub repo which is usually not desirable as it may contain experimental features or bugs. Favor using stable release versions of all gems unless there's a compelling reason not to.